docker/1.3.0/final/Dockerfile.cpu

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:16.04
+ARG framework_installable
+ARG py_version
+ARG framework_support_installable=sagemaker_mxnet_container-2.0.0.tar.gz
+
+# Validate that arguments are specified
+RUN test $framework_installable || exit 1 && \
+    test $py_version || exit 1
+
+WORKDIR /tmp
+
+RUN apt-get update && \
+    apt-get -y install libopencv-dev libopenblas-dev python python3 wget curl nginx
+
+# Symlink /usr/bin/python to the python version we're building for.
+RUN rm /usr/bin/python && ln -s "/usr/bin/python$py_version" /usr/bin/python
+
+# Install pip
+RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
+    /usr/bin/python get-pip.py && \
+    rm get-pip.py
```

Comment:
Install pip like this https://github.com/aws/sagemaker-tensorflow-container/blob/master/docker/1.11.0/final/py2/Dockerfile.cpu#L19